### ISSUE TITLE
A11y_30days_Welcome to 30 Days of Learning_Page Title: The Page title Name is present twice in the Title Element

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [microsoft/30daysof#182]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yTCS;#A11yMAS;#A11ySev3;#30days;#P3_30days_Dec22;#WCAG2.4.2;#DesktopWeb;#Win11;#ChromiumEdge;

### Environment Details:
Application Name: 30days
#URL: https://microsoft.github.io/30daysof/
Windows Version: Win11
Chromium Edge Version: 110.0.1556.0 (Official build) dev (64-bit)

### Repro Steps:

1. Hit the URL: https://microsoft.github.io/30daysof/
2. Inspect the page and check the title element in the code.
3. Observe that the title name is present twice inside the title element.

### Actual:
The Title element contains "Welcome to #30DaysOf Learning | #30DaysOf Learning". The name #30DaysOf Learning is present twice.

### Expected:
The title name should be present once inside the title element , i.e "Welcome to #30DaysOf Learning" .

### User Impact:
Screen reader user will be confused when they will hear the name twice.

Attachment:
![Screenshot (43)](https://user-images.githubusercontent.com/85863895/210195423-b59899ed-5138-4442-aa04-7f1002841c3e.png)

**Comment by Ashutoshsahoo164** [comment]
![182_Closed](https://user-images.githubusercontent.com/85863895/218798625-b60ada09-cfd1-452a-9ab0-0095370386c5.png)

Verified the bug . Issue is fixed. 
GithubTag: #Closed

**Comment by nitya** [comment]
@Ashutoshsahoo164 reopening this - can you verify and close?

**Comment by nitya** [comment]
NVM I see you closed it
